o/opentracing/redhat_ubi8/build.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+# Package       : opentracing
+# Version       : 1.6.0
+# Source repo   : https://github.com/opentracing/opentracing-cpp
+# Tested on     : RHEL_8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora
+#
+# Disclaimer: This script has been tested in root mode on given platform using
+#             the mentioned version of the package. It may not work as expected
+#             with newer versions of the package and/or distribution.
+#             In such case, please contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+# clone branch/release passed as argument, if none, use master
+
+if [ -z $1 ] || [ "$1" == "lasttestedrelease" ]
+then
+       # As on 19 Sep 2019, release 1.6.0was latest
+       BRANCH="--branch v1.6.0"
+else
+       BRANCH="--branch $1"
+fi
+
+echo "BRANCH = $BRANCH"
+
+# Clone Repository
+cd $HOME/
+git clone https://github.com/opentracing/opentracing-cpp
```

Comment:
Appears master is cloned at all times; probably $BRANCH usage was missed and you intended: -
`git clone $BRANCH https://github.com/opentracing/opentracing-cpp`

Reply:
Have updated per feedback, please review.
Marking this resolved.